core/tests/protractor_utils/forms.js

```diff
@@ -560,19 +560,20 @@ var CodeMirrorChecker = function(elem) {
    * currentLineNumber is the current largest line number processed,
    * scrollTo is the number of pixels from the top of the text that
    * codemirror should scroll to,
+   8 Pane specifies the codemirror Left or Right Pane which is to be scrolled.
    * compareDict is an object whose keys are line numbers and whose values are
    * objects corresponding to that line with the following key-value pairs:
    *  - 'text': the exact string of text expected on that line
    *  - 'highlighted': true or false, whether the line is highlighted
    *  - 'checked': true or false, whether the line has been checked
    */
   var _compareTextAndHighlightingFromLine = function(
-      currentLineNumber, scrollTo, compareDict) {
+      currentLineNumber, scrollTo, Pane, compareDict) {
```

Comment:
I don't think `Pane` is a good name for this parameter. Please rename something on the lines `codeMirrorPaneToScroll`. Also, please follow the camel case for JS variables.

Reply:
Done